tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/featured_case_study.html

```diff
@@ -0,0 +1,53 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% image featured_case_study_image fill-828x546 format-webp as featured_desktop_image %}
+{% image featured_case_study_image fill-1656x1092 format-webp as featured_desktop_image_retina %}
+{% image featured_case_study_image fill-860x646 format-webp as featured_mobile_image %}
+{% image featured_case_study_image fill-430x323 format-webp as featured_mobile_image_retina %}
```

Comment:
I think these last two need to swap so the larger image is used for retina

Reply:
D'oh! https://github.com/torchbox/torchbox.com/pull/60/commits/bf3d3ba09e3394f693a96c98095f2bfdacab5c98